docs/resources/kubernetes_cluster.md

```diff
@@ -173,12 +179,17 @@ In addition to the arguments listed above, the following additional attributes a
   - `id` -  A unique ID that can be used to identify and reference the node pool.
   - `actual_node_count` - A computed field representing the actual number of nodes in the node pool, which is especially useful when auto-scaling is enabled.
   - `nodes` - A list of nodes in the pool. Each node exports the following attributes:
-     + `id` -  A unique ID that can be used to identify and reference the node.
-     + `name` - The auto-generated name for the node.
-     + `status` -  A string indicating the current status of the individual node.
-     + `droplet_id` - The id of the node's droplet
-     + `created_at` - The date and time when the node was created.
-     + `updated_at` - The date and time when the node was last updated.
+    + `id` -  A unique ID that can be used to identify and reference the node.
+    + `name` - The auto-generated name for the node.
+    + `status` -  A string indicating the current status of the individual node.
+    + `droplet_id` - The id of the node's droplet
+    + `created_at` - The date and time when the node was created.
+    + `updated_at` - The date and time when the node was last updated.
+  - `taint` - A list of taints applied to all nodes in the pool. Each taint exports the following attributes:
```

Comment:
> I'd like some feedback about the attribute name: taint. Since this represents a list, I can see this being confusing and wanted to know how others feel about making this plural (would match the API also). Although, the singular tense makes the UI in terraform config files much clearer.

I think singular works fine here. As you mention it feels more natural in a Terraform config, e.g.:

```
    taint {
      key    = "workloadKind"
      value  = "database"
      effect = "NoSchedule"
    }

    taint {
      key    = "priority"
      value  = "high"
      effect = "NoSchedule"
    }
```

Maybe the solution is to reword the docs a bit?

```suggestion
  - `taint` - A block representing a taint applied to all nodes in the pool. Each taint exports the following attributes:
```